packages/service-clients/azure-client/package.json

```diff
@@ -137,7 +137,11 @@
 	},
 	"typeValidation": {
 		"broken": {
-			"RemovedClassDeclaration_AzureFunctionTokenProvider": {
+			"RemovedClassStatics_AzureFunctionTokenProvider": {
```

Comment:
The change mentioned in the PR description. @CraigMacomber am I right that this is expected because of your updates?

Reply:
Yes. This case was previously not tested, and had an preexisting break (same reason one below this is broken)